actors/builtin/market/market_actor.go

```diff
@@ -526,14 +514,12 @@ func (a Actor) CronTick(rt Runtime, params *adt.EmptyValue) *adt.EmptyValue {
 				if err != nil {
 					return xerrors.Errorf("failed to get cid for deal proposal: %w", err)
 				}
-				if err := pending.Delete(adt.CidKey(dcid)); err != nil {
-					builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to delete pending proposal")
-				}
+
+				builtin.RequireNoErr(rt, pending.Delete(adt.CidKey(dcid)), exitcode.ErrIllegalState, "failed to delete pending proposal")
```

Comment:
@anorth I think writing things inline like this is fairly difficult to read, we should probably at least break things like this into two lines.

Reply:
Yes, I agree. The `pending.Delete` call would be better on its own line.

Reply:
@anorth This has been fixed.